### PR TITLE
Content/ns-timeline-event

### PR DESCRIPTION
**Pull request by MekalaNagarajan-Centrica**
- [x] Timeline event example variants.
- [x] Timeline `step` variant example.
- [x] Status table.

### CODE REVIEW

File: src/content/docs/components/ns-timeline.mdx [hunk 1]
```diff
@@ -18,8 +18,14 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
 The timeline gives an overview of a process and can show the progression of a customer journey. Its purpose is to create situational awareness for the user and manage expectations.
 
 ### Examples
-
-<ExampleViewer type="standard" />
+<Tabs>
+  <TabItem label="Standard">
+    <ExampleViewer type="standard" />
+  </TabItem>
+  <TabItem label="Step">
```
Comment (andij) [inline src/content/docs/components/ns-timeline.mdx:25]: When selecting this tab, the height collapses.

- As it's across all Examples that have tabs. I've added a Bug item to the Docs backlog #473

File: src/components/specification.astro [hunk 2]
```diff
@@ -32,7 +32,10 @@ const events = compDetails?.events;
 
 { 
   properties?.length > 0 &&  (
-    <h4 id="attributes">Attributes</h4>
+    <h4 id="attributes">
+      Attributes
+      <a aria-hidden="true" tabindex="-1" href="#attributes"><span class="icon icon-link"></span></a>
```
Comment (andij) [inline src/components/specification.astro:37]: There's a slim chance that another heading on the page will be called **Attributes**. May I suggest making these more unique.

```suggestion
      <a aria-hidden="true" tabindex="-1" href="#specification-attributes"><span class="icon icon-link"></span></a>
```

File: src/components/specification.astro [hunk 3]
```diff
@@ -108,7 +111,10 @@ const events = compDetails?.events;
 
 {
   slots?.length > 0 && (
-    <h4 id="slots">Slots</h4>
+    <h4 id="slots">
+      Slots
+      <a aria-hidden="true" tabindex="-1" href="#slots"><span class="icon icon-link"></span></a>
```
Comment (andij) [inline src/components/specification.astro:116]: There's a slim chance that another heading on the page will be called **Slots**. May I suggest making these more unique.

```suggestion
      <a aria-hidden="true" tabindex="-1" href="#specification-slots"><span class="icon icon-link"></span></a>
```

File: src/components/specification.astro [hunk 4]
```diff
@@ -138,7 +144,10 @@ const events = compDetails?.events;
 
 {
   events?.length > 0 && (
-    <h4 id="events">Events</h4>
+    <h4 id="events">
+      Events
+      <a aria-hidden="true" tabindex="-1" href="#events"><span class="icon icon-link"></span></a>
```
Comment (andij) [inline src/components/specification.astro:149]: There's a slim chance that another heading on the page will be called **Events**. May I suggest making these more unique.

```suggestion
      <a aria-hidden="true" tabindex="-1" href="#specification-events"><span class="icon icon-link"></span></a>
```

File: src/content/docs/components/ns-timeline-event.mdx [hunk 5]
```diff
@@ -15,9 +15,38 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 <CompDetails name={componentName} />
 
+:::caution
+This component can only be placed within `ns-timeline`. Refer `ns-timeline` [specification](/components/ns-timeline#slots) for usage.
+:::
+
 ### Examples
+<Tabs>
```
Comment (andij) [inline src/content/docs/components/ns-timeline-event.mdx:23]: Are we able to include a step variant here?

Comment (andij) [inline src/content/docs/components/ns-timeline-event.mdx:23]: Whitespace

```suggestion
### Examples

<Tabs>
```

Comment (MekalaNagarajan-Centrica) [reply src/content/docs/components/ns-timeline-event.mdx:23]: Step variant is part of `ns-timeline`.

Comment (andij) [reply src/content/docs/components/ns-timeline-event.mdx:23]: I was referring to there not being an example of `<ns-timeline-event>` showing a number in these tabs. E.g., without `status` and without `icon`.

File: src/content/docs/components/ns-timeline.mdx [hunk 6]
```diff
@@ -18,8 +18,14 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
 The timeline gives an overview of a process and can show the progression of a customer journey. Its purpose is to create situational awareness for the user and manage expectations.
```
[the remark this answers is not in the document]
Comment (RobTobias123) [reply src/content/docs/components/ns-timeline.mdx:19]: I concur with this Drew.

[the remark this answers is not in the document]
Comment (MekalaNagarajan-Centrica) [reply src/content/docs/components/ns-timeline.mdx:19]: Slots in the specification table already cover the slot name and permitted tags for it.

File: src/content/docs/components/ns-tab.mdx [hunk 7]
```diff
@@ -17,6 +17,10 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 The `ns-tab` is the button that changes the content in the `ns-tabs` when clicked. It shows the user relevant content with a heading and an optional icon.
 
+:::caution
+This component can only be placed within `ns-tabs`. Refer [`ns-tabs` specification](/components/ns-tabs#specification-slots) for usage.
```
Comment (andij) [inline src/content/docs/components/ns-tab.mdx:21]: Please include the angle brackets and capitalise Specification:

```suggestion
This component can only be placed within `ns-tabs`. Refer [`<ns-tabs>` Specification](/components/ns-tabs#specification-slots) for usage.
```

File: src/content/docs/components/ns-timeline-event.mdx [hunk 8]
```diff
@@ -15,9 +15,39 @@ import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 <CompDetails name={componentName} />
 
+:::caution
+This component can only be placed within `ns-timeline`. Refer [`ns-timeline` specification](/components/ns-timeline#specification-slots) for usage.
```
Comment (andij) [inline src/content/docs/components/ns-timeline-event.mdx:19]: Please include the angle brackets and capitalise Specification:

```suggestion
This component can only be placed within `ns-timeline`. Refer [`<ns-timeline>` Specification](/components/ns-timeline#specification-slots) for usage.
```

File: src/content/docs/components/ns-timeline.mdx (hunk 6 above)
[the remark this answers is not in the document]
Comment (MekalaNagarajan-Centrica) [reply src/content/docs/components/ns-timeline.mdx:19]: This is more or less any other component with a slot. Also we have linked the dependency.

[the remark this answers is not in the document]
Comment (andij) [reply src/content/docs/components/ns-timeline.mdx:19]: So, if we are doing this, we should also hide `ns-expander`. Providing links to it from `ns-accordion`, `ns-skyline`, `ns-selector` and `ns-product-card`.

File: src/content/docs/components/ns-timeline-event.mdx (hunk 5 above)
Comment (andij) [reply src/content/docs/components/ns-timeline-event.mdx:24]: Okay, thanks for explaining. It is a bit of a shame that we're unable to show the `ns-timeline-event` in the state for the `step` variant.

**Review by andij — APPROVED** [review]

**Pull request merged**